ibmvmc/ibmvmc.c

```diff
@@ -655,12 +662,13 @@ static int ibmvmc_close(struct inode *inode, struct file *file)
 			return -EIO;
 		}
 
-		/* TODO locking? */
+		spin_lock_irqsave(&hmc->lock, flags);
```

Comment:
Was this change needed for some reason for the sleep in process context?  Or you just cleaned this up because of the TODO?